ros_sam/package.xml

```diff
@@ -50,20 +50,13 @@
   <!--   <doc_depend>doxygen</doc_depend> -->
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>cv_bridge</build_depend>
-  <build_depend>geometry_msgs</build_depend>
   <build_depend>rospy</build_depend>
```

Comment:
do build dependencies make sense for a python package?

Reply:
That is a good question and I don't know. Truthfully, I overload the `package.xml` files, because catkin is fickle